apostrophecms/apostrophe
# Pro 2235 pieces context menu

[PRO-2235](https://linear.app/apostrophecms/issue/PRO-2235/as-a-developer-i-can-configure-my-module-to-add-a-new-button-to-the)

## Purpose
Adds a configurable contextual menu to piece type modules, it allows to add multiple actions to a module.

## How to test
Add a moreMenu object to any piece type module, it should add it to your manager modal contextual menu.
Ex:
![image](https://user-images.githubusercontent.com/17548370/140059789-78327bca-1250-4296-a215-671d2310b610.png)


PS: I named it `moreMenu`, I don't really like this name but found it somewhere, if you have better naming for this do not hesitate.
